tests/Feature/MoviesTest.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Movie;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class MoviesTest extends TestCase
```

Comment:
You can create function that can create movie for you. Make your code more DRY! ;)